File: Project Euler/Problem_2/solution.py
def main():
    total = 0
    a = 1
    b = 2

    while(a <= 4000000):
        t = a + b
        if a%2 == 0:
            total = total + a
        a = b
        b = t

    print(total)

if __name__ == '__main__':
    main()

File: Project Euler/Problem_2/Solution.java
class Solution {
    public static void main(String[] args) {
		int total = 0;
		int a = 1;
		int b = 2;
		
		while (a <= 4000000) {
			int t = a + b;
			if (a % 2 == 0) total += a;
			a = b;
			b = t;
		}

		System.out.println(total);
    }
}